driver.cpp

```diff
@@ -447,6 +447,124 @@ namespace bitcoinfuzz
         }
     }
 
+    void Driver::PrivateToPublicTarget(std::span<const uint8_t> buffer) const
+    {
+        FuzzedDataProvider provider(buffer.data(), buffer.size());
+        if (buffer.size() < 32) { return; }
+
+        std::vector<uint8_t> privKeyBuffer = provider.ConsumeBytes<uint8_t>(32);
+        std::optional<std::string> last_response{std::nullopt};
+        std::string last_module_name;
+
+        for (auto &module : modules)
+        {
+            std::optional<std::string> res{module.second->private_to_public(privKeyBuffer)};
+            if (!res.has_value()) continue;
+            if (last_response.has_value()) {
+                if (*res != *last_response) {
+                    std::cout << "PrivateToPublic Target failed" << std::endl;
+                    std::cout << "Module: " << module.first << std::endl;
+                    std::cout << "Result: " << *res << std::endl;
+                    std::cout << "Module: " << last_module_name << std::endl;
+                    std::cout << "Result: " << *last_response << std::endl;
+                }
+                assert(*res == *last_response);
+            }
+
+            last_response = res.value();
+            last_module_name = module.first;
+        }
+    }
+
+    void Driver::SignCompactTarget(std::span<const uint8_t> buffer) const
+    {
+        FuzzedDataProvider provider(buffer.data(), buffer.size());
+        if (buffer.size() < 64) { return; }
+
+        std::vector<uint8_t> privKeyBuffer = provider.ConsumeBytes<uint8_t>(32);
+        std::vector<uint8_t> hashBuffer = provider.ConsumeBytes<uint8_t>(32);
+        std::optional<std::string> last_response{std::nullopt};
+        std::string last_module_name;
+
+        for (auto &module : modules)
+        {
+            std::optional<std::string> res{module.second->sign_compact(privKeyBuffer, hashBuffer)};
+            if (!res.has_value()) continue;
+            if (last_response.has_value()) {
+                if (*res != *last_response) {
+                    std::cout << "SignCompact Target failed" << std::endl;
+                    std::cout << "Module: " << module.first << std::endl;
+                    std::cout << "Result: " << *res << std::endl;
+                    std::cout << "Module: " << last_module_name << std::endl;
+                    std::cout << "Result: " << *last_response << std::endl;
+                }
+                assert(*res == *last_response);
+            }
+
+            last_response = res.value();
+            last_module_name = module.first;
+        }
+    }
+
+    void Driver::SignDerTarget(std::span<const uint8_t> buffer) const
+    {
+        FuzzedDataProvider provider(buffer.data(), buffer.size());
+        if (buffer.size() < 64) { return; }
+
+        std::vector<uint8_t> privKeyBuffer = provider.ConsumeBytes<uint8_t>(32);
```

Comment:
Please, use snake case for new vars.

```suggestion
        std::vector<uint8_t> privkey_buffer{provider.ConsumeBytes<uint8_t>(32)};
```